dcicutils/qa_utils.py

```diff
@@ -4,13 +4,27 @@
 
 import contextlib
 import datetime
+import time
 import io
 import os
 import pytz
 
 from .misc_utils import PRINT, ignored, Retry
 
 
+@contextlib.contextmanager
+def timed():
```

Comment:
I might either make this a class so the report can be customized by different methods or else make something that shows the info be an outboard function that becomes the default. e.g,
```
def show_elapsed_time(start, end):
    print('Elapsed: %s' % (end - start))
@contextlib.contextmanager
def timed(reporter=None):
    if reporter is None:
        reporter = show_elapsed_time
    ...
    finally:
        end = time.time()
        show_elapsed_time(end, start)
```

Alternatively the class-based way of doing it would just have that be a method call instead of a functional parameter.

The reason I care, though, is that I'm not sure print is always what you want. You might actually want to have it do logging of some kind, including possibly using that `debuglog` capability I just recently put in `dcicutils.miscutils`. But having it come out on stdout is sometimes disruptive.